plugin/src/sbt-test/sbt-native-image/agent/test

```diff
@@ -0,0 +1,5 @@
+$ absent hello3.obtained
+> example/nativeImageRunAgent
+> example/nativeImage
+> example/nativeImageRun
+$ must-mirror hello3.expected hello3.obtained
```

Comment:
How do you know that `hello3.obtained` was written by `nativeImageRun` insted of `nativeImageRunAgent`? Would it make sense to remove `hello3.obtained` before `example/nativeImageRun`?